brax/training/networks.py

```diff
@@ -82,48 +83,55 @@ def __call__(self, data: jnp.ndarray):
         hidden = self.activation(hidden)
     return hidden
 
+def canolicalize_obs_size(obs_size: types.ObservationSize) -> int:
```

Comment:
nit: "canonical" can be a quite overloaded term, let's call this `get_obs_state_size`